### PR TITLE
chore: add doc install with keink

### PR DESCRIPTION
**Pull request by Qaplagzy**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: docs/setup/install-with-keink.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# Deploying KubeEdge with Keink
```
Comment (Shelley-BaoYue) [inline docs/setup/install-with-keink.md:1]: add a head with title and sidebar_position ref to https://github.com/kubeedge/website/blob/master/docs/setup/install-with-binary.md?plain=1  

The sidebar_position determines the order of the file in the directory. I think it's better to put it after install-with-binary. How about you?  Accordingly, the sidebar_position number of the other files should be adjusted.  
image

File: docs/setup/install-with-keink.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+keink(represent for [KubeEdge](https://github.com/kubeedge/kubeedge) IN [kind](https://github.com/kubernetes-sigs/kind)) is a tool for running local KubeEdge clusters using Docker container "nodes".It allows for the easy deployment of a KubeEdge cluster on a single machine. This method is only used for trial kubeedge, please do not use it in a production environment.
+
+## Environment Preparation
+
+### KubeEdge Codes
+
+You need to download the KubeEdge codes locally. It's recommended to place it in the `$GOPATH` directory, as keink can automatically detect the KubeEdge source code in this directory. Alternatively, you can manually enter the code location.
```
Comment (Shelley-BaoYue) [inline docs/setup/install-with-keink.md:9]: I think we should draft a release for keink, users don't need clone keink code and build themselves. @fisherxu @wbc6080

Comment (wbc6080) [reply docs/setup/install-with-keink.md:16]: +1

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Review by Shelley-BaoYue — COMMENTED** [review]
/approve

**Pull request merged**